bin/oref0-setup.sh

```diff
@@ -281,12 +293,8 @@ elif [[ ${CGM,,} =~ "shareble" ]]; then
     fi
     sudo apt-get -y install bc jq libusb-dev libdbus-1-dev libglib2.0-dev libudev-dev libical-dev libreadline-dev python-dbus || die "Couldn't apt-get install: run 'sudo apt-get update' and try again?"
     echo Checking bluez installation
-    if ! bluetoothd --version | grep -q 5.37 2>/dev/null; then
```

Comment:
yes, cause we don't need need to copy that .conf for just using the Tethering, so it's already checking if shareble, so it should have already installed bluez, this will allow that copy of the .conf. otherwise it would not have copied it.
now does that cause a problem if someone is just re-running the script, I don't know. someone using shareble needs to test.